moban/engine_handlebars.py

```diff
@@ -77,7 +93,7 @@ def _apply_template(self, template, data, output):
         template_file = self.find_template_file(template)
         with open(template_file, "r") as source:
             if sys.version_info[0] < 3:
-                template = Compiler().compile(unicode(source.read())) # noqa
+                template = Compiler().compile(unicode(source.read()))  # noqa
```

Comment:
unrelated change - create an issue for these pep issues, and schedule it for a beginner